{
  "uid" : "2278fd653eca91fe",
  "name" : "test_estimate_bridge_stablecoin_to_not_native[Avalanche (AVAX)-Avalanche (AVAX)]",
  "fullName" : "tests.api_tests.tests_de_bridge#test_estimate_bridge_stablecoin_to_not_native",
  "historyId" : "cd55dddf3ccd411676d97be946be7c93",
  "time" : {
    "start" : 1686067357725,
    "stop" : 1686067358194,
    "duration" : 469
  },
  "status" : "failed",
  "statusMessage" : "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method=\\\"decimals()\\\", data=\\\"0x\\\", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)\"}.",
  "statusTrace" : "src_nets = 'Avalanche (AVAX)', dst_nets = 'Avalanche (AVAX)'\n\n    @pytest.mark.parametrize(\"src_nets\", networks)\n    @pytest.mark.parametrize(\"dst_nets\", networks)\n    def test_estimate_bridge_stablecoin_to_not_native(src_nets, dst_nets):\n        src_net = de_bridge_keys_data()[src_nets].get('net')\n        dst_net = de_bridge_keys_data()[dst_nets].get('net')\n        src_stable_coin = de_bridge_keys_data()[src_nets].get('stableCoin')\n        dst_not_native_token = de_bridge_keys_data()[dst_nets].get('notNativeToken')\n>       DeBridge().EstimateBridge().estimate_bridge_stablecoin_to_not_native(src_net=src_net, dst_net=dst_net,\n                                                                             src_stable_coin=src_stable_coin,\n                                                                             dst_not_native_token=dst_not_native_token)\n\ntests/api_tests/tests_de_bridge.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge.EstimateBridge object at 0x7fe74d8c0490>\nsrc_net = 'avalanche', dst_net = 'avalanche'\nsrc_stable_coin = '0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d'\ndst_not_native_token = '0xe896cdeaac9615145c0ca09c8cd5c25bced6384c'\n\n    def estimate_bridge_stablecoin_to_not_native(self, src_net, dst_net, src_stable_coin, dst_not_native_token):\n        response = requests.get(\n            f\"{self.url}?srcNet={src_net}&srcTokenAddress={src_stable_coin}\"\n            f\"&srcTokenAmount=10000&dstNet={dst_net}&dstTokenAddress={dst_not_native_token}\")\n        #logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        #logger.info(response.text)\n        print(\"Logs\")\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method=\\\"decimals()\\\", data=\\\"0x\\\", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)\"}.\n\nserviceCalls/debrdige_service.py:54: AssertionError",
  "flaky" : false,
  "newFailed" : false,
  "beforeStages" : [ ],
  "testStage" : {
    "status" : "failed",
    "statusMessage" : "AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \n{\"ok\":false,\"data\":null,\"error\":\"call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method=\\\"decimals()\\\", data=\\\"0x\\\", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)\"}.",
    "statusTrace" : "src_nets = 'Avalanche (AVAX)', dst_nets = 'Avalanche (AVAX)'\n\n    @pytest.mark.parametrize(\"src_nets\", networks)\n    @pytest.mark.parametrize(\"dst_nets\", networks)\n    def test_estimate_bridge_stablecoin_to_not_native(src_nets, dst_nets):\n        src_net = de_bridge_keys_data()[src_nets].get('net')\n        dst_net = de_bridge_keys_data()[dst_nets].get('net')\n        src_stable_coin = de_bridge_keys_data()[src_nets].get('stableCoin')\n        dst_not_native_token = de_bridge_keys_data()[dst_nets].get('notNativeToken')\n>       DeBridge().EstimateBridge().estimate_bridge_stablecoin_to_not_native(src_net=src_net, dst_net=dst_net,\n                                                                             src_stable_coin=src_stable_coin,\n                                                                             dst_not_native_token=dst_not_native_token)\n\ntests/api_tests/tests_de_bridge.py:36: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ \n\nself = <serviceCalls.debrdige_service.DeBridge.EstimateBridge object at 0x7fe74d8c0490>\nsrc_net = 'avalanche', dst_net = 'avalanche'\nsrc_stable_coin = '0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d'\ndst_not_native_token = '0xe896cdeaac9615145c0ca09c8cd5c25bced6384c'\n\n    def estimate_bridge_stablecoin_to_not_native(self, src_net, dst_net, src_stable_coin, dst_not_native_token):\n        response = requests.get(\n            f\"{self.url}?srcNet={src_net}&srcTokenAddress={src_stable_coin}\"\n            f\"&srcTokenAmount=10000&dstNet={dst_net}&dstTokenAddress={dst_not_native_token}\")\n        #logger.info(f\"Отправлен запрос на роут - {response.url}\")\n        #logger.info(response.text)\n        print(\"Logs\")\n>       assert response.status_code == 200, f\"Ожидался status = {200}, \" \\\n                                            f\"пришел status = {response.status_code}. \" \\\n                                            f\"Тело ответа: \\n{response.text}.\"\nE       AssertionError: Ожидался status = 200, пришел status = 400. Тело ответа: \nE       {\"ok\":false,\"data\":null,\"error\":\"call revert exception [ See: https://links.ethers.org/v5-errors-CALL_EXCEPTION ] (method=\\\"decimals()\\\", data=\\\"0x\\\", errorArgs=null, errorName=null, errorSignature=null, reason=null, code=CALL_EXCEPTION, version=abi/5.7.0)\"}.\n\nserviceCalls/debrdige_service.py:54: AssertionError",
    "steps" : [ ],
    "attachments" : [ {
      "uid" : "a877cbbd10375df7",
      "name" : "stdout",
      "source" : "a877cbbd10375df7.txt",
      "type" : "text/plain",
      "size" : 5
    } ],
    "parameters" : [ ],
    "stepsCount" : 0,
    "attachmentsCount" : 1,
    "shouldDisplayMessage" : true,
    "hasContent" : true
  },
  "afterStages" : [ ],
  "labels" : [ {
    "name" : "parentSuite",
    "value" : "tests.api_tests"
  }, {
    "name" : "suite",
    "value" : "tests_de_bridge"
  }, {
    "name" : "host",
    "value" : "fv-az593-404"
  }, {
    "name" : "thread",
    "value" : "2509-MainThread"
  }, {
    "name" : "framework",
    "value" : "pytest"
  }, {
    "name" : "language",
    "value" : "cpython3"
  }, {
    "name" : "package",
    "value" : "tests.api_tests.tests_de_bridge"
  }, {
    "name" : "resultFormat",
    "value" : "allure2"
  } ],
  "parameters" : [ {
    "name" : "dst_nets",
    "value" : "'Avalanche (AVAX)'"
  }, {
    "name" : "src_nets",
    "value" : "'Avalanche (AVAX)'"
  } ],
  "links" : [ ],
  "hidden" : false,
  "retry" : false,
  "extra" : {
    "severity" : "normal",
    "retries" : [ ],
    "categories" : [ {
      "name" : "Product defects",
      "matchedStatuses" : [ ],
      "flaky" : false
    } ],
    "tags" : [ ]
  },
  "source" : "2278fd653eca91fe.json",
  "parameterValues" : [ "'Avalanche (AVAX)'", "'Avalanche (AVAX)'" ]
}